README.md

```diff
@@ -15,36 +15,37 @@ The analytics interfaces has to be configured in Webpack configuration file thro
 
 ```javascript
 new webpack.DefinePlugin({
-  __analytics: JSON.stringify({
```

Comment:
what is the reason you've moved this `JSON.stringify` into child-properties, and only to few of them? 🤔  :)

Reply:
webpack.DefinePlugin has a tricky rules :D https://webpack.js.org/plugins/define-plugin/ 
to allow to use tree shaking isEnabled should be `true`. When we stringify object in isEnabled you have string and webpack to use tree shaking need boolen :D